build.gradle.kts

```diff
@@ -31,6 +31,7 @@ dependencies {
     implementation("io.quarkus:quarkus-hibernate-orm")
     implementation("io.quarkus:quarkus-hibernate-validator")
     implementation("io.smallrye.reactive:mutiny-kotlin")
+    implementation("com.google.code.gson:gson:2.8.8")
```

Comment:
You already have Jackson in the project and therefore you don't need Gson. 

Remove this.